recipes/pjsip/2.8/conanfile.py

```diff
@@ -0,0 +1,182 @@
+import os
+import stat
+import glob
+import shutil
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+_openSSL = "OpenSSL"
+
+# based on https://github.com/conan-community/conan-ncurses/blob/stable/6.1/conanfile.py
+class PjsipConan(ConanFile):
+    name = "pjsip"
+    version = "2.8"
+    license = "GPL2"
+    homepage = "https://github.com/pjsip/pjproject/"
+    description = "PJSIP is a free and open source multimedia communication library written in C language implementing standard based protocols such as SIP, SDP, RTP, STUN, TURN, and ICE."
+    url = "https://github.com/jens-totemic/conan-pjsip"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "SSL": [True, False],
+               # when using little endian on armv7, we can enable neon for pjsip
+               # this setting replaces host arm-linux-gnueabihf with armv7l-linux-gnueabihf
+               # if our current arch is set to any armv7
+               "armv7l": [True, False],
+               # the default echo cancellation algo is SPEEX_AEC
+               # in order to prefer WEBRTC_AEC, we can disable it
+               "disableSpeexAec": [True, False],
+               "fPIC": [True, False]}
+    # if no OpenSSL is found, pjsip might try to use GnuTLS
+    default_options = {"shared": False, "SSL": True, "armv7l": True, "disableSpeexAec":True, "fPIC": True}   
+    generators = "cmake"
+    exports = "LICENSE"
+    _autotools = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.shared
+
```

Comment:
сишная библиотека, стоит добавить на всякий случай (по аналогии с например openssl):
```suggestion

    def configure(self):
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
        
```